client/wfnews-war/src/main/angular/src/app/components/report-of-fire/compass-page/rof-compass-page.component.html

```diff
@@ -6,10 +6,10 @@
         <img alt="Compass Hand" [src]="compassHandUrl"  class="compass-hand-image"/>
     </div>
     <div class="compass-heading" id="compass-heading">{{ heading }}</div>
-    <div class="location">{{ currentLat }},{{ currentLong }}</div>
+    <div class="location" id="location">{{ currentLat }},{{ currentLong }}</div>
     <div class="rof-button-container">
       <button class="rof-button" mat-flat-button (click)="previous()">Previous</button>
-        <button class="rof-button-primary" mat-flat-button (click)="next()">Continue</button>
+        <button class="rof-button-primary" mat-flat-button (click)="next(); confirmHeading()">Continue</button>
```

Comment:
In this case, you can likely move the call to `next()` directly into the `confirmHeading()`. Cleaner that way. Make sure to try/catch the `confirmHeading()` so it doesn't puke and fail the next

Reply:
Sounds good, updated accordingly